libs/partners/milvus/langchain_milvus/vectorstores/milvus.py

```diff
@@ -549,15 +593,25 @@ def _create_collection(
                     max_length=65_535,
                 )
             )
-        # Create the vector field, supports binary or float vectors
-        if self._is_sparse_embedding:
-            fields.append(FieldSchema(self._vector_field, DataType.SPARSE_FLOAT_VECTOR))
-        else:
-            fields.append(
-                FieldSchema(
-                    self._vector_field, infer_dtype_bydata(embeddings[0]), dim=dim
+
+        embeddings_functions: List[EmbeddingType] = (
+            [self.embeddings] if not self._is_hybrid else self.embeddings
+        )
+        for i, vectors in enumerate(embeddings):
+            dim = len(vectors[0])
```

Comment:
This assumes that `embeddings` are `list[list]` which is correct for calls that go through `add_texts`.
If it is always the case, we could change the func signature, otherwise we should verify it here (and wrap with `list` if it not)

Reply:
Will take a look again, but changing the signature is a problem as I must keep it backward compatible

Reply:
In that case we must handle here the case of a single list of embedding